README.md

```diff
@@ -6,33 +6,12 @@
 
 Ruby client for the [Unleash](https://github.com/Unleash/unleash) feature management service.
 
-- [Supported Ruby Interpreters](#supported-ruby-interpreters)
-- [Installation](#installation)
```

Comment:
I realized that GitHub now has a built-in TOC for readmes so I think we can get rid of these artisinal ones. They take up quite a bit of space, and I prefer when we can just get to the point right away

![Screenshot 2024-09-26 at 12 36 02](https://github.com/user-attachments/assets/9149a49b-0ec1-44a4-bb3b-c6647dfddf82)

Reply:
YESSS